AccessCheckoutSDK/AccessCheckoutSDK/validation/model/pan/PanValidationFlow.swift

```diff
@@ -30,52 +46,44 @@ class PanValidationFlow {
 
         let cardNumberPrefix = String(sanitisedCardNumber.prefix(12))
 
-        let hasChanged = cardNumberPrefix != lastCheckedPanPrefix
-
-        if hasChanged {
+        if cardNumberPrefix != lastCheckedPanPrefix {
             lastCheckedPanPrefix = cardNumberPrefix
 
-            let globalBrand = panValidationStateHandler.getCardBrand()
-
-            // currently logs out card bin lookup result for debugging purposes
-            // will call PanValidationStateHandler to handle updating merchant delegeate with returned card brands
             cardBinService.getCardBrands(
-                globalBrand: globalBrand,
+                globalBrand: self.panValidationStateHandler.getGlobalBrand(),
                 cardNumber: cardNumberPrefix
             ) { result in
                 switch result {
                 case .success(let cardBrands):
-                    NSLog("Card BIN lookup succeeded: \(cardBrands)")
+                    if self.panValidationStateHandler.areCardBrandsDifferentFrom(
+                        cardBrands: cardBrands)
+                    {
+                        // The global brand always appear first in the list of brands returned by OUR CardBinService class
+                        self.updateCvcValidationRule(for: cardBrands.first)
+                        self.panValidationStateHandler.updateCardBrandsIfChanged(
+                            cardBrands: cardBrands)
+                    }
                 case .failure(_):
-                    NSLog("Card BIN lookup failed:")
+                    NSLog("Card BIN lookup failed")
```

Comment:
Is this useful - review and review if error needs to be logged

Reply:
this code is not reachable as the CardBinService never calls the callback in case of failure